lib/pbench/server/indexing_tarballs.py

```diff
@@ -437,8 +439,15 @@ def sighup_handler(*args):
                                         else States.QUARANTINED
                                     )
                                     Metadata.remove(dataset, Metadata.REINDEX)
+                                    Metadata.create(
+                                        dataset=dataset,
+                                        key=Metadata.INDEX_MAP,
+                                        value=json.dumps(ptb.index_map),
```

Comment:
I've considered changing the Dataset Metadata value column to `JSON` which would be more general and make the code slightly neater. That would touch a few other places, though.